### DISCUSSION TITLE
Кнопка возврата в цикле

### DISCUSSION THREAD

**Discussion by Leo-ui-528** [totumonline/totum-ru-issues-and-discussions#247] [Как это сделать?] [answered]
Подскажите, как сделать кнопку возврата в из таблицы в цикле в основную таблицу цикла?

**Comment by k1erk** [comment]
=: linkToTable(table: 'table_name'; target: "self";)

**Comment by Leo-ui-528** [comment]
спасибо

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Там же стандартная кнопка есть. Она вернет с учетом Префильтров если переход был из Циклов в Цикл.

<img width="1137" alt="Снимок экрана 2022-10-25 в 14 16 17" src="https://user-images.githubusercontent.com/55755565/197759185-3c7d4c15-88f1-4844-92b0-1ed796fea481.png">

**Reply by erpic-ru** [reply]
Простите за оффтоп, но дарк-тема выглядит просто отлично!

**Reply by ttm-alexeypolunin** [reply]
Типа не зря когда-то в художку ходил :) Уже на GitHub — обновляйте PRO.

На следующей неделе сделаем возможность подключать кастомные CSS и на основе CSS-ки для темной темы можно будет нарулить свой корпоративный лук. Но сразу скажу — это дофига непросто.

<img width="305" alt="Снимок экрана 2022-10-25 в 15 32 49" src="https://user-images.githubusercontent.com/55755565/197773887-79872285-82c5-4976-ad80-bd36f65aa194.png">

**Comment by Leo-ui-528** [comment]
Да, согласен дарк тема выглядит гораздо лучше )Подскажите какую ксс редактировать на сервере может быть, и вставить свой логотип можно?

**Reply by ttm-alexeypolunin** [reply]
В MIT в том числе

<img width="1322" alt="Снимок экрана 2022-11-01 в 19 24 07" src="https://user-images.githubusercontent.com/55755565/199284798-989f5b4d-d1f9-4d88-8b82-6eb0ee2dd121.png">

**Reply by ttm-alexeypolunin** [reply]
Кастомные CSS можно будет на следующей неделе загружать, но только в PRO

**Comment by Leo-ui-528** [comment]
Спасибо

**Comment by Leo-ui-528** [comment]
![image](https://user-images.githubusercontent.com/75322878/199483826-eddf208a-f677-4833-9a3a-d31c2d2b13a0.png)

Текс заголовка меняется, картинка нет, подскажите почему. Может к картинке определенные требования не соблюдены мною?

**Reply by ttm-alexeypolunin** [reply]
Полез разбираться и понял, что она обязательно должна быть `.png`

Поправлю как-нибудь для следующих установок.

**Reply by ttm-alexeypolunin** [reply]
Да, поломалось ограничение по mime-type — он по идее не должен давать загрузить `jpeg`.

**Comment by Leo-ui-528** [comment]
![image](https://user-images.githubusercontent.com/75322878/199494916-2b9920b2-a06a-477e-91b8-8bb696aef254.png)
Так тоже не грузит

**Reply by ttm-alexeypolunin** [reply]
обновить страницу с обновлением кэша конешн

**Comment by Leo-ui-528** [comment]
Получилось, спасибо)
